Source/Core/VideoBackends/Vulkan/VulkanContext.cpp

```diff
@@ -834,6 +837,13 @@ void VulkanContext::InitDriverDetails()
     driver = DriverDetails::DRIVER_UNKNOWN;
   }
 
+#ifdef __APPLE__
+  // Vulkan on macOS goes through MoltenVK, and is not susceptible to the same bugs
+  // as the vendor's native Vulkan drivers. We use a different driver fields to
+  // differentiate MoltenVK.
+  driver = DriverDetails::DRIVER_MOLTENVK;
```

Comment:
Let's rewrite the comment and call it `DriverDetails::PORTABILITY` or `DriverDetails::VK_PORTABILITY`.